src/main/java/com/google/sps/data/GovPostReq.java

```diff
@@ -34,12 +49,15 @@
     //Enable DoOutput to send request content and write content to output stream
     con.setDoOutput(true);
 
+    String desiredDate = inputDate.format(DateTimeFormatter.ofPattern("yyyy-MM-dd"));
+    System.out.println(LocalDate.now());
+
     //Create the Request Body 
     final String jsonInputString = "{" +
       "\"Parameters\": [ \"AQI\", ]," +
       "\"Sites\": [ ]," +
-      "\"StartDate\": \"2021-01-18\"," +
-      "\"EndDate\": \"2021-01-19\"," + 
+      String.format("\"StartDate\": \"%s\",", desiredDate) +
+      String.format("\"EndDate\": \"%s\",", desiredDate) + 
```

Comment:
startDate & endDate is the same day, will it work?

Reply:
yep, it works

Reply:
you can try it out if you want, the unfilled data will be returned as -1

Reply:
ok